client/templates/hangout/hangout-list.js

```diff
@@ -21,7 +21,8 @@ Template.hangoutList.onRendered(function() {
 
     instance.loadCompletedHangouts = function() {
       var now = new Date();
-      return Hangouts.find({'end': {$lt : now}}, {sort: { start: -1 }});
+      var now_in_min = now.setMinutes(now.getMinutes())
+      return Hangouts.find({'end': {$lt : now_in_min}}, {sort: { start: -1 }});
```

Comment:
I think think we need to have checks for both somehow, since only hangouts which have been manually ended will have their `end` time in minutes. This might generally be the case, but any seeded hangouts will now not show up.

Can we change the format of hangouts that are ended manually to have a regular timestamp? Then I think the previous code would work.